templates/arm-aks-template.json

```diff
@@ -134,6 +141,13 @@
           {
             "name": "[variables('nodepoolName')]",
             "count": "[parameters('nodeCount')]",
+            "maxCount": "[parameters('nodeMaxCount')]",
+            "minCount": "[variables('nodeMinCount')]",
```

Comment:
I think this still needs maintaining when rebuilding a cluster, you want it to come back with enough nodes and not continuously scaling trying to schedule everything?